### ISSUE TITLE
5. Changing Content is missing WCAG Techniques section

### ISSUE THREAD

**Issue by kengdoj** [atbcb/ICTTestingBaseline#332]
was this intentional? are there no techniques to list?

Instead of leaving out this section, marking none would be better.

**Comment by kengdoj** [comment]
Completed.

added WCAG 2.0 Techniques heading and [F20: Failure of Success Criterion 1.1.1 and 4.1.2 due to not updating text alternatives when changes to non-text content occur](https://www.w3.org/WAI/WCAG21/Techniques/failures/F20.html)

Preview: https://federalist-78ccea55-7a06-447c-aabf-bd620b207b74.app.cloud.gov/preview/atbcb/icttestingbaseline/nextupdateafter301/05Changing/
